.github/workflows/release.yml

```diff
@@ -33,5 +35,49 @@ jobs:
           name: nugets
           path: nugets/*
           retention-days: 1
-      - name: Deploy to MyGet
-        run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}
+     #- name: Deploy to MyGet
+       #run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}       
+      - name: Build SelfContained
+        run: >
+          dotnet publish src/Tool/Particular.EndpointThroughputCounter.csproj -r win-x64
```

Comment:
Not sure this is the best way to handle this, and I think there are some redundant/unneeded things in this list of properties.

Reply:
Our goal was a single exe. ReadyToRun may not be needed. TargetLatestRuntimePath is questionable, but the rest is to package everything into the single exe including pdb, and all other binaries.